tsl/test/expected/cagg_joins-12.out

```diff
@@ -52,7 +52,49 @@ INSERT INTO devices values (1, 'thermo_1', 'Moscow'), (2, 'thermo_2', 'Berlin'),
 CREATE TABLE devices_dup AS SELECT * FROM devices;
 CREATE VIEW devices_view AS SELECT * FROM devices;
 --Create a cagg with join between a hypertable and a normal table
--- with equality condition on inner join type
+-- with equality condition on inner join type and realtime aggregation enabled
+CREATE MATERIALIZED VIEW conditions_summary_daily_realtime
+WITH (timescaledb.continuous) AS
+SELECT time_bucket(INTERVAL '1 day', day) AS bucket,
+   AVG(temperature),
+   MAX(temperature),
+   MIN(temperature),
+   name
+FROM conditions, devices
+WHERE conditions.device_id = devices.device_id
+GROUP BY name, bucket;
+NOTICE:  refreshing continuous aggregate "conditions_summary_daily_realtime"
+HINT:  Use WITH NO DATA if you do not want to refresh the continuous aggregate on creation.
+\d+ conditions_summary_daily_realtime
+              View "public.conditions_summary_daily_realtime"
+ Column |  Type   | Collation | Nullable | Default | Storage  | Description 
+--------+---------+-----------+----------+---------+----------+-------------
+ bucket | date    |           |          |         | plain    | 
+ avg    | numeric |           |          |         | main     | 
+ max    | integer |           |          |         | plain    | 
+ min    | integer |           |          |         | plain    | 
+ name   | text    |           |          |         | extended | 
+View definition:
+ SELECT _materialized_hypertable_3.bucket,
+    _materialized_hypertable_3.avg,
+    _materialized_hypertable_3.max,
+    _materialized_hypertable_3.min,
+    _materialized_hypertable_3.name
+   FROM _timescaledb_internal._materialized_hypertable_3 _materialized_hypertable_3
+  WHERE _materialized_hypertable_3.bucket < COALESCE(_timescaledb_internal.to_date(_timescaledb_internal.cagg_watermark(3)), '-infinity'::date)
+UNION ALL
+ SELECT time_bucket('@ 1 day'::interval, conditions.day) AS bucket,
+    avg(conditions.temperature) AS avg,
+    max(conditions.temperature) AS max,
+    min(conditions.temperature) AS min,
+    devices.name
+   FROM conditions,
+    devices
+  WHERE conditions.device_id = devices.device_id AND devices.device_id >= COALESCE(_timescaledb_internal.to_date(_timescaledb_internal.cagg_watermark(3)), '-infinity'::date)
+  GROUP BY devices.name, (time_bucket('@ 1 day'::interval, conditions.day));
+
```

Comment:
Could you insert data into the real-time part of the CAGG and do a `SELECT` on the CAGG to show that real-time aggregation works?